excel/src/main/java/tech/tablesaw/io/xlsx/XlsxReader.java

```diff
@@ -60,14 +60,42 @@ public static void register(ReaderRegistry registry) {
 
   @Override
   public Table read(XlsxReadOptions options) throws IOException {
-    List<Table> tables = readMultiple(options);
+    List<Table> tables = readMultiple(options, true);
     if (tables.isEmpty()) {
```

Comment:
You probably don't need this check anymore since there are other checks that would cover it as well

Reply:
:-)